make-release.sh

```diff
@@ -89,6 +89,11 @@ releaseCheCode() {
     invokeAction che-incubator/che-code "Release Che Code" "34764281" "version=${CHE_VERSION}"
 }
 
+
+releaseDevworkspaceGenerator() {
+    invokeAction eclipse-che/che-devfile-registry "Release Che Devworkspace Generator" "TODO->WORKFLOW-ID" "version=${CHE_VERSION}"
```

Comment:
I may be a simple, country, code monkey, but this seems like it'll fail as `TODO->WORKFLOW-ID` dos not seem to be a valid workflow ID.

Reply:
before merging this PR, I'll update it with live ID, once che-devfile-registry PR will be merged

Reply:
updated with actual ID